1_fetch/src/fetch_data.R

```diff
@@ -0,0 +1,18 @@
+# fetch model RMSE data
+
+## preliminaries 
+library(readr)
+library(sbtools)
+
+## fetch data
+### default saves as '1_fetch/out/model_RMSEs.csv'
+
+fetch_data<-function(output_path = '1_fetch/out/model_RMSEs.csv'){
+  
+  # fetch data and write file
+  item_file_download('5d925066e4b0c4f70d0d0599', names = 'me_RMSE.csv', destinations = output_path, overwrite_file = TRUE)
+  print(paste('RMSE data written to ', output_path))
```

Comment:
Just a suggestion for the print statements: 1) perhaps use `message()` instead of `print()`? 2) when I do this kind of thing within a function (have it print out something related to the status), I also tend to add a `verbose = TRUE` argument to the function so that I can toggle it on or off. Would you consider something like that?